rmc/utils/generic.py

```diff
@@ -377,6 +377,15 @@ def process_vep(ht: hl.Table, filter_csq: bool = False, csq: str = None) -> hl.T
         ht, vep_root="vep", syn=False, canonical=True, filter_empty=True
     )
 
+    logger.info("Filtering to non-outlier transcripts...")
```

Comment:
This additional filtering step for non-outlier transcripts should be reflected in the description of the resources that are generated with `process_vep` i.e. `filtered_exomes` and `processed_context`.